### ISSUE TITLE
changelog for same branch

### ISSUE THREAD

**Issue by frank-w** [marvinpinto/actions#69]
Hi,
I got a changelog always for differences compared to default branch. Is there any way to get changelog for changes in same branch till last release?

I have a kernel repo where i build each LTS version (4.14, 4.19, 5.4, 5.10) on push

https://www.github.com/frank-w/BPI-R2-4.14

If this is not possible,can i disable the changelog feature?

**Comment by marvinpinto** [comment]
Could what you're asking for be accomplished with https://github.com/marvinpinto/actions/issues/68 ?

**Comment by frank-w** [comment]
I don't really want use additional tags for this...i create tags after build.

How do you think it can solve the branch-specific changelog?

I've noticed the changelog-creation takes some time and sometimes (nondefault branch) too long so that the assets are not complete till email-notification

**Comment by marvinpinto** [comment]
I think the automatic releases action will probably not be helpful for you. It works primarily on tags and I'm not sure how I would reword it to work with base branches and such.

**Comment by frank-w** [comment]
Currently your action is the best i've found,which can start multiole commands and upload multiple files.

How can i archive the difference to the latest tag (if i filter tag with branchname)

**Comment by marvinpinto** [comment]
Hmm, I suppose it might be doable to provide a SHA or tag, instead of a tag in #68. I think this could then be used as the backstop to search for changelogs. Would that get you what you want?

**Comment by frank-w** [comment]
can you give me an example how i can set the previous release-tag?

i create a new tag like this on each release:

`automatic_release_tag: "CI-BUILD-${{ env.BRANCH }}-${{ env.KERNELVER }}-${{ env.DT }}"`

example: CI-BUILD-5.10-main-5.10.25-2021-03-23_0847

maybe i can filter out this based on the env.BRANCH variable

my full config for same branch:
https://github.com/frank-w/BPI-R2-4.14/blob/CI-BUILD-5.10-main-5.10.25-2021-03-23_0847/.github/workflows/build.yml
